service/internal/releaseversion/release_versions.go

```diff
@@ -0,0 +1,117 @@
+package releaseversion
+
+import (
+	"context"
+
+	releasev1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1"
+	"github.com/giantswarm/k8sclient"
+	"github.com/giantswarm/microerror"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"github.com/giantswarm/cluster-operator/service/controller/key"
+
```

Comment:
This should not pass CI since we want 3 blocks but the automation is not smart enough with that as it seems. 

```suggestion
```